recipes/docker-image-guide.md

```diff
@@ -133,7 +135,7 @@ CMD [ "node", "index.js" ]
   By copying source code files after dependency installation, we are only
   re-executing those steps that come after that step, including that step.
 
-10. `RUN npm ci --only=production`
+10. `RUN npm ci`
```

Comment:
Does it make sense to also describe here that NODE_ENV needs to be set to production in order to install only production dependencies?